test/ios_coverage_test.sh

```diff
@@ -182,6 +233,13 @@ ios_unit_test(
     test_coverage_manifest = "coverage_manifest.txt",
     runner = "@build_bazel_rules_apple//apple/testing/default_runner:ios_xctestrun_ordered_runner",
 )
+
+ios_ui_test(
+    name = "test_coverage_ui_test",
+    deps = [":coverage_ui_test_lib"],
+    minimum_os_version = "15",
+    test_host = ":coverage_app",
+)
```

Comment:
Is this using the xctestrun runner you're mutating or the other one? I figure you actually need to provide the `runner` attribute like I think you had at one point.

Reply:
I confused myself here. Will fix! Thanks for the callout!